finalfusion_inspector/metadata.py

```diff
@@ -94,11 +93,11 @@ def switchEmbeddings(self, embeddings):
     def updateMetadata(self):
         self._metadata = []
 
-        if not isinstance(self.embeddings.metadata, str):
+        if not isinstance(self.embeddings.metadata, dict):
```

Comment:
`Metadata` subclasses `dict`, but since it's Python, someone could throw other types in there, to...

Reply:
True, then they won't see metadata I guess? Good punishment ;).